src/quantum/index.jsx

```diff
@@ -107,14 +117,14 @@ export default class QuantumIndex extends React.Component {
         title: 'Performance Tests',
         rows: [
           <PerfherderGraphContainer
-            title='Page load (tp5)'
```

Comment:
@sarah-clements , @klahnakoski  
There is a linting error here in all components. 
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-key.md 

All components which have missing `key` attribute are giving errors.  I'm not sure what should I add in `key` attribute. 

Also, is this [keys](https://github.com/mozilla-frontend-infra/firefox-health-dashboard/blob/7d522aaa530fd9479337f909f5ac967520570712/src/quantum/index.jsx#L60) attribute similar to the `key`? This component is also giving errors. 

What to add in key?

Reply:
I think in this case you can add a key that corresponds to the title and is unique, so `<PerfherderGraphContainer title="Page load (tp5)" key="page-load-tp5" ...>`.  React needs the `key` property so that it can track when components move around in the virtual DOM - for example if an item is removed from an array and all of the others shift by one position.  Using a name allows React to see that the item now in position 5 in the array used to be in position 6, for example.